.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,68 @@
+name: Generic issue
+description: Report an issue, a suggestion, SQL only fix.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this form! Please make sure to fill all fields, including the Title above.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Description of the problem or issue here.
+        If this is a crash, post the crashlog (upload to https://gist.github.com/).
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: |
+        Tell us what should happen instead.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce the problem
+      description: |
+        1. Step 1 include entries of affected creatures / items / quests with a link to the relevant wowhead or https://tcubuntu.northeurope.cloudapp.azure.com/aowow/ page.  
+        2. Step 2
+        3. Step 3
+    validations:
+      required: true
+  - type: dropdown
+    id: branch
+    attributes:
+      label: Branch
+      options:
+        - 3.3.5
+        - master
+    validations:
+      required: true
+  - type: textarea
+    id: commit
+    attributes:
+      label: TC rev. hash/commit
+      description: |
+        Copy the result of server debug command (if you need to run it from client get prat addon)
```

Comment:
```suggestion
        Copy the output of server console (when used ".server debug" command) or from game client (by using "Prat" addon)
```

Reply:
should `when used ".server debug" command` be `when using ".server debug" command` ?

Reply:
using sounds better :)

Reply:
Suggested version is a bit confusing

Reply:
As the PR has been merged, feel free to open a new PR changing the text. Both versions sounds a bit confusing to me, maybe we will come up with something that is even clearer :)